stories/components/Tab.stories.mdx

```diff
@@ -0,0 +1,315 @@
+               <div name="networking">NestedTab content.</div>
+            </Tab>
+          </Tabbed>
+        </Tab>
+      </Tabbed>
+    `
+});
+
+export const AddRemoveDefault = (args, { argTypes, events }) => {
+  const [_, updateArgs] = useArgs();
+  return {
+    components: { Tabbed, Tab },
+  props:      Object.keys(argTypes),
+  data: () => {
+    return {
+      value: args?.tabs || [],
+    };
+  },
+  methods: {
+    update(value) {
+      let new_item = [];
+      let item_object = {};
+      const i = this.value.length + 1;
+      item_object.name = `Tab${i}`;
+      item_object.content = `Tab content ${i}`;
+      new_item.push(item_object);
+      this.value = [...this.value, ...new_item];
+      updateArgs({
+        tabs: this.value
+      });
+    },
+    remove(value) {
+      this.value.splice(value, 1);
+      updateArgs({
+        tabs: this.value
+      });
+    }
+  },
+  template:   `
+    <Tabbed v-bind="$props">
+      <Tab
+        v-for="g in value"
+        :key="g.name"
+        :name="g.name"
+        :label="g.name"
+      >
+      <div>{{ g.content }}
+      <br/>
+      <button type="button" class="btn-sm role-link" @click="remove($event)">
+        <i class="icon icon-minus icon-lg" /> Remove
+      </button>
+      </div>
+      </Tab>
+      <template #tab-row-extras>
+        <li class="tablist-controls">
+          <button type="button" class="btn-sm role-link" @click="update($event)">
+            <i class="icon icon-plus icon-lg" /> Add
+          </button>
+        </li>
+      </template>
+    </Tabbed>
+    `,
+  };
+};
+
+
+# Tab
+
+Tabs are used to separate information into logical sections and to quickly navigate between them.
+
+### Description
+- Use tabs when there is a large amount of content that can be separated.
+- Use tabs to make the content accessible without reloading the page or compromising on space.
+- Write short tab labels. Try not to use more than two words for a tab label.
+
+- Secondary tabs never appear without a parent set of primary tabs.
+- The number of secondary tabs inside vertical tabs should be as small as possible – not more than six are recommended.
+<br/>
+<br/>
+
+### Types
+
+<br/>
+
+#### Default/Horizontal Tabs
+- Horizontal tabs are best used with ten or fewer tabs. This maintains an uncluttered UI and reduces cognitive load for users.
+- Write short tab labels. Especially with a large number of tabs and little horizontal screen space.
+- Stick to only one row of tabs. Use tab navigation when the content categories fit in a single row.
+
+<Canvas>
+  <Story
+    name="Default"
+    args={{
+      useHash: false,
+    }}>
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+#### Side/Vertical Tabs
+- Vertical tabs provide better scalability for a large number of tabs (ten and more).
+
+<Canvas>
+   <Story
+    name="Vertical"
+    args={{
+      useHash: false,
+      sideTabs: true,
+    }}>
+    {Template.bind({})}
+  </Story>
+</Canvas>
+
+
+#### Nested
```

Comment:
We had an issue to identify exactly cases where we had 3 nested tabs, as defined in the issue: horizontal, vertical, vertical.
Could you please add it?

Reply:
As we do not have currently this case, we opted to exclude it.